wpilibc/src/main/native/include/frc/XboxController.h

```diff
@@ -35,72 +35,94 @@ class XboxController : public GenericHID {
   XboxController& operator=(XboxController&&) = default;
 
   /**
-   * Get the X axis value of the controller.
-   *
-   * @param hand Side of controller whose value should be returned.
+   * Get the left X axis value of the controller.
```

Comment:
Whilst we're splitting these, we might as well make this super-clear what these do.

```suggestion
   * Get the X axis value of the left stick on the controller.
```